include/roaring/roaring_array.h

```diff
@@ -1,5 +1,8 @@
 #ifndef INCLUDE_ROARING_ARRAY_H
 #define INCLUDE_ROARING_ARRAY_H
+#ifdef __cplusplus
+extern "C" {
+#endif
```

Comment:
I needed to add another extern "C" block since this file is referred to from within the C++ code